doc/source/examples/incompressible-flow/2d-transient-flow-around-cylinder/2d-transient-flow-around-cylinder.rst

```diff
@@ -0,0 +1,281 @@
+
+.. code-block:: text
+
+  gnuplot -c "./postprocess.gnu" "./Re200"
+
+where ``./postprocess.gnu`` is the path to the provided script and ``./Re200`` is the path to the directory that contains the simulation results (specified in the ``simulation control`` subsection). The figure, named ``CL-CD.png``, is outputted in the directory ``./Re200``.
+
+.. note::
+
+  Gnuplot is a command-line plotting tool supporting scripting. It can be downloaded `here <http://www.gnuplot.info/>`_. The script provided for this example works for the version 5.4 of Gnuplot.
+
+.. image:: images/CL-CD.png
+    :alt: CD and CL evolution in time
+    :align: center
+    :name: CD-CL
+
+Using the fast Fourier transform (FFT) of the CL for the last 100 seconds, we can obtain the frequency :math:`f_v` at which the vortices are shed :
+
+.. image:: images/cylinderFFT.png
+    :alt: Strouhal Number
+    :align: center
+    :name: Strouhal
+
+This corresponds to the frequency at which the peak of amplitude appears in the FFT : :math:`f_v = 0.2`. From this result, we can obtain the Strouhal number, :math:`S_t = 0.2`, using the equation presented above. The python script used to obtain the FFT is available in the example folder and is launched in the same directory using the following command:
+
+.. code-block:: text
+
+  python ./postprocess.py ./Re200
+
+where ``./postprocess.py`` is the path to the provided script  and ``./Re200`` is the path to the directory that contains the simulation results (specified in the ``simulation control`` subsection). The figure, named ``cylinderFFT.png``, is outputted in the directory ``./Re200``.
+
+The obtained values of the drag and lift coefficients as well as the Strouhal number are compared to some results of the literature :
+
+.. list-table::
+   :widths: 20 20 20 20
+   :header-rows: 1
+
+   * - Study
+     - :math:`C_D`
+     - :math:`C_L`
+     - :math:`S_t`
+   * - Lethe example
+     - 1.392 :math:`\pm` 0.048
+     - -0.006 :math:`\pm` 0.072
+     - 0.2
+   * - Lethe Sharp [2]
+     - 1.395 :math:`\pm` 0.047
+     - :math:`\pm` 0.071
+     - 0.2
+   * - Braza et al. [3]
+     - 1.400 :math:`\pm` 0.050
+     - :math:`\pm` 0.075
+     - 0.2
+
+
+Using Paraview the following velocity and pressure fields can be visualized in time:
+
+.. image:: images/cylinderVelocity.gif
+    :alt: Velocity profile
+    :align: center
+    :name: velocity
+
+.. image:: images/cylinderPressure.gif
+    :alt: Pressure profile
+    :align: center
+    :name: pressure
+
+Possibilities for extension
+----------------------------
+- Study the vortex shedding of other bluff bodies.
+- Increase the Reynolds number to study a completely turbulent wake and the drag crisis phenomenon.
+- Repeat the same example in 3D for a cylinder/sphere and study the effect on the drag and lift forces.
```

Comment:
I would add something like:
Investigate the impact of the time-step and the time-stepping scheme (e..g sdirk 3 or bdf 3)